scripts/index.html

```diff
@@ -6,6 +6,15 @@
         <h1 id="page-title">
           Meeting Integration Tests Sample
         </h1>
+        <form id="adapter">
+          <fieldset>
+            <legend>JS SDK Adapter</legend>
+            <input id="credentials" placeholder="Access Token" type="password" />
+            <button id="create">Create</button>
+            <button id="destroy">Destroy</button>
```

Comment:
```suggestion
            <button id="destroy-adapter">Destroy</button>
```

Just to make it more specific if/when things start to expand

Reply:
These are used as nested id's in the implementation so it is the "destroy" element within the "adapter".

Reply:
There is no such thing as "nested IDs" in HTML